### ISSUE TITLE
[FEATURE] Create Start Menu in Unity

### ISSUE THREAD

**Issue by beanbeanjuice** [beanbeanjuice/back-to-the-jungle#15]
## Feature Description
This depends on #5. Create and lay out the start menu. It is up to you if you just want to create the UI/Scene and have no functionality, and want to delegate the functionality to a separate issue for the input team. Just make sure you create a separate issue, citing this issue as a dependency.

___

## Checklist

- [x] I have checked for similar issues.
- [x] This is a feature request, not an improvement request, bug report, or security vulnerability report.
- [x] I have assigned myself or someone else to this issue.
